src/components/hoc/withExtendedTouchable.tsx

```diff
@@ -53,15 +53,15 @@ export function withExtendedTouchable<T extends TouchableOpacityProps | Touchabl
         // Async onPress. Always show busy form until promise resolves
         if (typeof result?.then === 'function') {
           setIsAwaiting(true)
-          result.catch(e => showError(e)).finally(() => setIsAwaiting(false))
+          result.catch(e => showDevError(e)).finally(() => setIsAwaiting(false))
```

Comment:
I would leave this one as `showError`. Suppose the user presses a button to do something, and an error occurs. If we hide the error, it will feel like the button is broken, because there is zero feedback. The user will just keep mashing the button until they give up and decide the button is useless, because nothing happens. If we *do* show the error, then at least they can call support and say, "When I hit the button to frob the wiznits, I get a red drop-down saying not enough woozits."